src/agent.h

```diff
@@ -43,7 +44,7 @@ class Agent : public rcsc::PlayerAgent {
   virtual FieldEvaluator::ConstPtr getFieldEvaluator() const;
 
   // Get the current game status
-  static hfo::status_t getGameStatus(const rcsc::AudioSensor& audio_sensor,
+  static std::vector<int> getGameStatus(const rcsc::AudioSensor& audio_sensor,
```

Comment:
The comment above this method should change to describe the contents of the vector<int> that is being returned.